linux/test/README.md

```diff
@@ -5,19 +5,16 @@ This directory contains Dockerfiles for testing the installation walkthroughs.
 
 For example:
 
-    ./docker-build.sh ubuntu1404
-    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_ubuntu1404
+    ./docker-build.sh ubuntu1404_nginx
+    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_ubuntu1404_nginx
 
-    ./docker-build.sh centos6
-    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6
+    ./docker-build.sh centos6_apache22
+    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6_apache22
 
 See `docker run --help` for more information on these and other options
 for running docker images.
 
-Ubuntu 14.04 using apache needs the web configuration to be modified to
-enable the OMERO web server to run, see below.
-
-Centos 7 cannot be tested in this way as systemd doesn't fully work, see below.
+CentOS 7 cannot be tested in this way as systemd doesn't fully work, see below.
```

Comment:
@ximenesuk I was planning to do in the centos6 ius PR 
the same applies to the new parameter for web. 
but if you are happy to review the changes in this PR even if it covers the other two I have fine with that